webroot/results/statistics.php

```diff
@@ -39,26 +39,28 @@ function showContent () {
     file_put_contents( 'generated/statistics.log', "$logMessage\n", FILE_APPEND );
     echo "<p>$logMessage</p>";
   }
-  
+
   #--- Show the cache
   echo file_get_contents( 'generated/statistics.cache' );
 }
 
 #----------------------------------------------------------------------
 function showResults () {
 #----------------------------------------------------------------------
-  
```

Comment:
nit: in general, it's nice if your editor doesn't trim all trailing whitespace, as it causes big diffs like this.

Don't bother fixing this, though. It's not worth your time =)